pkg/cmd/cli/install/install.go

```diff
@@ -130,11 +128,6 @@ func (o *Options) BindFlags(flags *pflag.FlagSet) {
 	flags.BoolVar(&o.DefaultSnapshotMoveData, "default-snapshot-move-data", o.DefaultSnapshotMoveData, "Bool flag to configure Velero server to move data by default for all snapshots supporting data movement. Optional.")
 	flags.BoolVar(&o.DisableInformerCache, "disable-informer-cache", o.DisableInformerCache, "Disable informer cache for Get calls on restore. With this enabled, it will speed up restore in cases where there are backup resources which already exist in the cluster, but for very large clusters this will increase velero memory usage. Default is false (don't disable). Optional.")
 	flags.BoolVar(&o.ScheduleSkipImmediately, "schedule-skip-immediately", o.ScheduleSkipImmediately, "Skip the first scheduled backup immediately after creating a schedule. Default is false (don't skip).")
-	flags.IntVar(&o.MaintenanceCfg.KeepLatestMaitenanceJobs, "keep-latest-maintenance-jobs", o.MaintenanceCfg.KeepLatestMaitenanceJobs, "Number of latest maintenance jobs to keep each repository. Optional.")
```

Comment:
IMO We think we should keep support for these flags

Reply:
This change is a bit different from the current compliance guidance.
So I'm open to adding those parameters back, but there are some reasons to remove them.
* `--keep-latest-maintenance-jobs` was deleted due to we see several issues reporting the maintenance jobs are not cleaned according to the setting. Please check this issue for more information https://github.com/vmware-tanzu/velero/issues/7923. After the dual-mode log introduced, we prepare to use this on maintenance job in v1.16. So in the end, we don't need to keep the maintenance job anymore.

Reply:
* The other paramers: The deletion reason is the function was still available, but moved to the new ConfigMap. If keeping them in the `velero server`, this makes the value assignment a little complicated to understand. Since we already have the `global` section as the default value, giving user this CLI parameters as other defaults seems not necessary.

Reply:
@shubham-pampattiwar 
After discussion, I added those parameters back.